docs/standard/serialization/binary-serialization-core.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Binary serialization in .NET Core"
+description: Learn which types support binary seralization in .NET Core.
+ms.date: "10/08/2017"
+ms.prod: ".net"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "binary serialization, .net core serialization"
+  - "serialization, cross-framework"
+author: "ViktorHofer"
+ms.author: "mairaw"
+---
+# Supported types for binary serialization in .NET Core
+
+ .NET Core supports binary serialization with a subset of types. You can see the list of supported types in the [next section](#serializable-types). The defined set of types are guaranteed to be serializable between .NET Framework 4.5.1 and later versions and .NET Core 2.0 and later versions. Other .NET implementations, such as Mono, aren't officially supported but should also be working.
+
+## Serializable types
+
+- <xref:System.AdjustmentRule?displayProperty=fullName>   
+- <xref:System.AggregateException?displayProperty=fullName>   
+- <xref:System.Array?displayProperty=fullName>   
+- <xref:System.ArraySegment%601?displayProperty=fullName>   
+- <xref:System.Attribute?displayProperty=fullName>   
+- <xref:System.Boolean?displayProperty=fullName>   
+- <xref:System.Byte?displayProperty=fullName>   
+- <xref:System.Char?displayProperty=fullName>   
+- <xref:System.Collections.ArrayList?displayProperty=fullName>   
+- <xref:System.Collections.BitArray?displayProperty=fullName>   
+- <xref:System.Collections.Comparer?displayProperty=fullName>   
+- <xref:System.Collections.DictionaryEntry?displayProperty=fullName>   
+- <xref:System.Collections.DictionaryNode?displayProperty=fullName>   
```

Comment:
Correct. It isn't exposed in the ref assemblies therefore we can remove the type from the list as we don't want people compiling against the src assemblies. The type is public in the src because of .NET Native Toolchain. I missed that. Thanks